common/src/main/java/com/box/l10n/mojito/DateTimeUtils.java

```diff
@@ -0,0 +1,75 @@
+package com.box.l10n.mojito;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.Date;
+
+/**
+ * Util class to provide Date/DateTime utility methods such as converters.
+ *
+ * @author hylston.barbosa
+ */
+public class DateTimeUtils {
```

Comment:
Suggestion: I think this util is more about the Joda migration. The goal is not to have yet anothr DateTimeUtils but more like isolating all the call site that a migrated in this commit. When everything is identified as working well, we could just inline the method.

Method names could be pretty explicit, for example: `fromDateToZonedDate` could be `newDateTimeOfJavaDate()` and the utility class name called `JodaMigrationUtils` or something along those lines